python-regression/RunLocalTests.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+printf "Begin local testing\n"
+printf "Available Tests: 
+    \tMachine 1: General API testing\n\t\tMinimum 2 nodes (nodeA,nodeB)\n\t\tDB: https://s3.eu-central-1.amazonaws.com/iotaledger-dbfiles/dev/testnet_files.tgz\n 
```

Comment:
it's not trivial that I need to run a node.
and when that is clear, it's not clear what INI / cli args are needed, the DB is not enough.